cms/templates/maintenance/_force_publish_course.html

```diff
@@ -0,0 +1,80 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="wrap-instructor-info studio-view maintenance-form">
+    <form class="form-create" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+        <div class="wrapper-form">
+            <fieldset>
+                <ol class="list-input">
+                    <li class="field text required" id="field-course-name">
+                        <label for="course-id">${_('Course ID')}</label>
+                        <input id="course-id" type="text" name="course-id" aria-required="true"
+                               value="${form_data['course_id']}"
+                               placeholder="${_('e.g. edX+DemoX+Demo_Course')}"/>
```

Comment:
I would like to see this placeholder text either moved to another location that is visible on screen all the time, or duplicated there.  Placeholder text disappears once the person starts typing.  In this particular example, the user would probably like to have access to the placeholder value once they start typing.  Can we put it below the text field in a ```<span>```?  The ```<input>``` would then require an aria-describedby attribute that references the IDref of the span.